### ISSUE TITLE
reference table, broken icons

### ISSUE THREAD

**Issue by allefeld** [zypA13510/octicons-webfont#13]
I made a reference table as an HTML document to be more easily able to pick the right HTML entity references for a given icon. I've attached it here ([octicons-webfont.html.gz](https://github.com/zypA13510/octicons-webfont/files/3902990/octicons-webfont.html.gz)) because I thought it might be a useful addition to the package documentation.

While viewing this in current Chrome (78.0.3904.108), I noticed that some icons are not correctly rendered.

![image](https://user-images.githubusercontent.com/1738353/69831364-f2347900-1220-11ea-833b-ed7390e8faa0.png)

As far as I can see, that applies to:
broadcast, circle-slash, clock, dashboard, dependent, eye-closed, file-pdf, file-zip, git-branch/merge/pull-request/action, history, kebab-horizontal/vertical, mark-github, person, primitive-dot-stroke, question, radio-tower, repo-forked, report, request-changes, rss, search, skip, sync, unmute, unverified, verified, workflow-all.

The same holds in Firefox 68.2.0esr and Atom 1.41.0.

Given that the original authors have abandoned webfont-support in primer/octicons, I think your repository can be extremely useful, but only if these errors are fixed.

**Comment by zypA13510** [comment]
Yes, I have also observed these issues with rendering. I guess they have something to do with the unmaintained `grunt-webfont` and its outdated dependencies, or the parameters [here](https://github.com/zypA13510/octicons-webfont/blob/master/tasks/config/webfont.yml). It could take some time to investigate. If you have any interesting discovery please let me know.

**Edit:** List of broken icons as of v1.5.0: (out of 204 total, 43 broken, 3 with minor issue)
- archive
- beaker
- broadcast
- circle-slash
- clock
- dashboard
- dependent
- eye-closed
- file-pdf
- file-zip
- git-branch
- git-compare
- git-merge
- git-pull-request
- github-action
- heart-outline
- history
- infinity
- internal-repo
- issue-closed
- issue-opened
- issue-reopened
- kebab-horizontal
- kebab-vertical
- mark-github
- pencil*
- person
- play
- primitive-dot-stroke
- project
- question
- radio-tower
- repo-forked
- report*
- request-changes*
- rocket
- rss
- search
- skip
- smiley
- sync
- unmute
- unverified
- verified
- workflow-all
- workflow
